chain/chain/src/store.rs

```diff
@@ -390,7 +389,7 @@ impl ChainStore {
         }
     }
 
-    pub fn owned_store(&self) -> Arc<Store> {
+    pub fn owned_store(&self) -> Store {
```

Comment:
This feels redundant now, as one can `.store().clone()`

Reply:
Fixed.